lldb/source/Commands/CommandObjectSource.cpp

```diff
@@ -1170,10 +1171,41 @@ class CommandObjectSourceList : public CommandObjectParsed {
           if (m_options.num_lines == 0)
             m_options.num_lines = 10;
           const uint32_t column = 0;
+
+          // Headers aren't always in the DWARF but if they have
+          // executable code (eg., inlined-functions) then the callsite's
+          // file(s) will be found. So if a header was requested and we got a
+          // primary file (ie., something with a different name), then look thru
+          // its support file(s) for the header.
+          lldb::SupportFileSP found_file_sp =
```

Comment:
And now get the file from `sc.line_entry.file_sp`

Reply:
done